examples/gno.land/r/sys/validators/v3/init.gno

```diff
@@ -0,0 +1,15 @@
+package validators
+
+import (
+	"std"
+
+	"gno.land/p/nt/poa"
+)
+
+func init() {
+	// The default valset protocol is PoA
+	vp = poa.NewPoA()
+
+	// Explicitly set the initial set
+	std.SetParamStrings(valsetPrevKey, []string{})
```

Comment:
It seems safer to simply check if the key exists from sdk, as we cannot practically force this contract to be uploaded or used.

Reply:
I'm not sure I get what you mean -- the initial state needs to be aligned between the Realm and chain in order for the diff to be valid and not rejected (cause a panic in the SDK)

Reply:
What if validators/v3 isn't loaded or reloaded? We should not consider this a requirement. Instead, we need to ensure that a chain without validators/v3 is still functional.